packages/x-components/src/x-modules/scroll/components/__tests__/scroll.spec.ts

```diff
@@ -29,37 +31,41 @@ function renderBaseIdScroll({
     }
   );
 
-  const wrapper = wrapperContainer.findComponent(BaseIdScroll);
+  const wrapper = wrapperContainer.findComponent(Scroll);
   const scrollElement: HTMLElement = wrapperContainer.find(
     getDataTestSelector('base-scroll')
   ).element;
   jest.spyOn(scrollElement, 'clientHeight', 'get').mockImplementation(() => clientHeight);
   jest.spyOn(scrollElement, 'scrollHeight', 'get').mockImplementation(() => scrollHeight);
 
+  /* Because the scroll mixins uses a $nextTick in the mounted hook, we have to add a wait here
+  so it has really finish mounting */
+  await wrapper.vm.$nextTick();
+
   return {
     wrapper,
     scrollElement,
-    async scroll(options: { to: number; durationMs: number }) {
-      scrollElement.scrollTop = options.to;
+    async scroll({ to, durationMs }) {
+      scrollElement.scrollTop = to;
       wrapper.trigger('scroll');
-      jest.advanceTimersByTime(options.durationMs);
+      jest.advanceTimersByTime(durationMs);
       await wrapper.vm.$nextTick();
     }
   };
 }
 
-describe('testing Base Scroll Component', () => {
+describe('testing Scroll Component', () => {
   beforeAll(jest.useFakeTimers);
   afterEach(jest.clearAllTimers);
```

Comment:
Same here as the other comment, if you remove this afterEach everything works the same.

Reply:
Same answer, this is just to ensure that each test is executed isolated and there are no pending calls from other tests.